assets/js/components/Accordion/Accordion.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+import { Disclosure, Transition } from '@headlessui/react';
+import { EOS_KEYBOARD_ARROW_DOWN } from 'eos-icons-react';
+
+import classNames from 'classnames';
+
+function Accordion({
+  className,
+  header,
+  headerClassnames,
+  withHeader = false,
```

Comment:
Can you rename this just to `header`? Since this accepts a JSX node I think it's better that way. Default can be `false` or just `undefined`